src/gui/editors/PianoRoll.cpp

```diff
@@ -202,10 +202,10 @@ PianoRoll::PianoRoll() :
 	signalMapper = new QSignalMapper( this );
 	m_semiToneMarkerMenu = new QMenu( this );
 
-	QAction * markSemitoneAction = new QAction( tr("Mark/unmark current semitone"), this );
-	QAction* markScaleAction = new QAction( tr("Mark current scale"), this );
-	QAction* markChordAction = new QAction( tr("Mark current chord"), this );
-	QAction* unmarkAllAction = new QAction( tr("Unmark all"), this );
+	QAction *markSemitoneAction = new QAction( tr("Mark/unmark current semitone"), this );
+	QAction *markScaleAction = new QAction( tr("Mark current scale"), this );
+	QAction *markChordAction = new QAction( tr("Mark current chord"), this );
+	QAction *unmarkAllAction = new QAction( tr("Unmark all"), this );
```

Comment:
This answered the question I were going to ask https://stackoverflow.com/questions/15212126/difference-between-char-var-and-char-var